### ISSUE TITLE
Bug: running eslint --init generates a .eslint.json file with parserOptions.ecmaversion set to 13 

### ISSUE THREAD

**Issue by oscar-bartman** [eslint/eslint#15159]
### Environment

Node version: v14.17.5
npm version: 6.14.14
Local ESLint version: 7.32.0
Global ESLint version: --
Operating System: Ubuntu Linux

### What parser are you using?

Default (Espree)

### What did you do?

<details>
<summary>Configuration</summary>

```
{
    "env": {
        "commonjs": true,
        "es2021": true,
        "node": true
    },
    "extends": [
        "standard"
    ],
    "parserOptions": {
        "ecmaVersion": 13
    },
    "rules": {
    }
}
```
</details>

```js
function hello() {
  console.log('hello')
}
```
Configuration is result from running `npx eslint --init` after installing eslint locally in an empty project.

Eslint shows an error at the first line of function shown saying `Parsing error: Invalid ecmaVersion.`

### What did you expect to happen?

When running `npx eslint --init` I expect that it generates an `.eslintrc` file with valid options. According to documentation, supported options for `parserOptions.ecmaVersion` are [3, 5 (default), 6, 7, 8, 9, 10, 11, or 12  ](https://eslint.org/docs/user-guide/configuring/language-options#specifying-parser-options). 

### What actually happened?

`npx eslint --init` generated an `.eslintrc` file with `parserOptions.ecmaVersion` set to 13.  

### Participation

- [X] I am willing to submit a pull request for this issue.

### Additional comments

Would love to help if anything. Just not sure where to start.

**Comment by aladdin-add** [comment]
thanks for the report@ 13 is an allowed value in eslint v8(but the docs was not updated somehow, should be fixed by #15162 )

**Comment by vicasas** [comment]
@aladdin-add What is the status? I see that #15162 was merged but at least from VS Code the warning still continues.

**Comment by aladdin-add** [comment]
@vicasas we are working on a new cli `@eslint/create-config` to replace `eslint --init`.  it will be fixed in the new repo, please follow https://github.com/eslint/eslint/issues/14768 .

**Comment by vicasas** [comment]
@aladdin-add From now on should we use `npx @eslint/create-config` to initialize eslint? I wonder since in the eslint docs still appears `npx eslint --init`

**Comment by aladdin-add** [comment]
yes, I think the issue has been fixed in the new package, and the docs will be updated in https://github.com/eslint/eslint/issues/14768(likely to be merged soon).
